packages/fxa-content-server/app/scripts/templates/partial/device-being-paired.mustache

```diff
@@ -1,8 +1,8 @@
-<div class="client {{ deviceType }}">
-    {{#deviceName}}<h2 class="device-name">{{deviceName}}</h2>{{/deviceName}}
-    <div class="family-os">{{#t}}%(family)s on %(OS)s{{/t}}</div>
+<div class="mt-9">
+    {{#deviceName}}<h2 class="mb-5 text-base">{{deviceName}}</h2>{{/deviceName}}
 
-    <div class="location">
+    <p class="text-xs" id="family-os">{{#t}}%(family)s on %(OS)s{{/t}}</p>
+    <p class="text-xs" id="location">
```

Comment:
Oh, this is mildly interesting.
I wonder if there is any chance this could conflict w/ the native `window.location`.
IIUC, when you use `id` on a DOM element, it adds some handle in the `window` scope as well.
For example, on the https://accounts.firefox.com/ page, we have this DOM element:

```html
<button id="use-logged-in" class="cta-primary cta-xl use-logged-in" type="submit">Sign in</button>
```

And in DevTools I can use this:

```text
window["use-logged-in"]
>> <button id="use-logged-in" class="cta-primary cta-xl use-logged-in" type="submit">
```

And I can inspect the `window.location` like so:
```text
window.location
>> Location https://accounts.firefox.com/signin
```

Maybe a non-issue since I can add an `id="location"` to a DOM element and then use `window.location` it still removes the Location object and not the DOM object. Probably a non-issue since we only seem to access the `id` as `#location` in the tests below. 🤷

Reply:
That's a very interesting note, thanks for sharing. I think in this instance, it's okay, because yeah, just using it for testing here and I can't think of an edgecase where this could get wonky. 🤔

Btw, kind of a side note but, I've been opting for `id` over `data-testid` when moving us away from non-Tailwind class names (and using `-testid` would've addressed your note), since in the hierarchy of testing in React `testing-library`, going by `data-testid` is at the very bottom of what's recommended. I figure when we rewrite all of this in React, we're going to be rewriting all of our tests anyway, and not using `data-testid` is kind of a nod to take a look at how it should be done rather than blindly copying over the `data-testid` attribute.

Reply:
You had me at _"rewrite all of this in React"_. 😍